src/removeStopwords.js

```diff
@@ -0,0 +1,38 @@
+// SUGGESTIONS: refacture ino shouldDisplay
+// and this function take a word and the language, then it returns true or false, and this function should be used wherever you are trying to display the words
+
+import {  STOPWORDS } from './constants';
+
+const filterWords = (wordCountObject, wordsToFilterOut) => {
+  for (var key in wordCountObject) {
+    var containsWord = (wordsToFilterOut.indexOf(key) > -1);
+    if (containsWord) {
+      delete wordCountObject[key];
+    }
+  }
+  return wordCountObject;
+}
+
+export const removeStopwords = (wordCountObject, language) => {
+  var stopwords = STOPWORDS[language];
```

Comment:
right now this function depends on (tightly coupled with) the actual words. so in your tests you have to use one of these words from the original stopwords. 

to avoid this behavior, we could do one of: 
- Mock `STOPWORDS` during the test (which is a bit complicated) 
- as `STOPWORDS` is a dependency that we can inject to this function and in tests we can inject different "simplified" words. 

for example: 

```js
export const createRemoveStopwords =(stopwords)=> (wordCountObject, language) => {
// logic here .. 
}
```

and then we export another function like that: 

```js
export const removeStopwords(STOPWORDS);
```

and when we test, we use custom/simple stop words: 

```js
+import { createRemoveStopwords } from './removeStopwords';

it( 'should remove stopords', () => {
    let stopWords = {'en': ['word1', 'stopword2', 'thirdword']}
    let mockedWords = { 'word1': 1, 'stays': 1, "'stopword2": 1,"'thirdword":1};
    let mockedLanguage = 'en';
    let expectedResult = {
      'stays': 1
    };

    let result = removeStopwords( mockedWords, mockedLanguage );
    expect( result ).toEqual( expectedResult );

  } );
} );
```

Reply:
I do not really understand what you mean by this:

> right now this function depends on (tightly coupled with) the actual words. so in your tests you have to use one of these words from the original stopwords.